app/classes/query/modules/lookup_names.rb

```diff
@@ -86,16 +86,16 @@ def find_exact_name_matches(vals)
   def find_matching_names(name)
     parse = Name.parse_name(name)
     name2 = parse ? parse.search_name : Name.clean_incoming_string(name)
-    matches = Name.where(search_name: name2) if parse&.author.present?
-    matches = Name.where(text_name: name2) if matches.empty?
+    matches = Name.unscoped.where(search_name: name2) if parse&.author.present?
+    matches = Name.unscoped.where(text_name: name2) if matches.empty?
```

Comment:
Is there a functional reason for adding `.unscoped` here?  If the only motivation is performance have you actually tested the performance of the query with and without?  I'm guessing this is unnecessary and just makes the code more complicated.